src/main/java/mekanism/common/item/ItemAtomicDisassembler.java

```diff
@@ -144,7 +148,9 @@ public boolean onBlockStartBreak(ItemStack itemstack, BlockPos pos, EntityPlayer
                     player.world.playEvent(WorldEvents.BREAK_BLOCK_EFFECTS, coord.getPos(), Block.getStateId(state));
                     player.world.setBlockToAir(coord.getPos());
                     block2.breakBlock(player.world, coord.getPos(), state);
-                    block2.dropBlockAsItem(player.world, coord.getPos(), state, 0);
+                    if(!player.capabilities.isCreativeMode) {
```

Comment:
The player can't be in creative here as it is checked at the top of the method.

Reply:
oh ok ill change that than